### ISSUE TITLE
Localised descriptions for jobs and factions

### ISSUE THREAD

**Issue by Peptide90** [Vault-Overseers/nuclear-14#65]
Upstream PR 13157 introduced job and department description tooltips.

Once this is merged into the stable branch we'll need to add these for our factions and roles.

Easy job for any new contributor as it's just localising the descriptions, look at the upstream PR for reference.
